examples/11.1.0/docs/angular/demo/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "angular-demo",
+  "private": true,
+  "version": "0.0.0",
+  "homepage": "https://handsontable.com/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/handsontable/handsontable.git"
+  },
+  "bugs": {
+    "url": "https://github.com/handsontable/handsontable/issues"
+  },
+  "author": "Handsoncode <hello@handsontable.com>",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve --port 8080",
+    "build": "ng build",
+    "test": "node spec/support/jasmine.config.js"
+  },
+  "dependencies": {
+    "@angular/animations": "11.2.7",
+    "@angular/common": "11.2.7",
+    "@angular/compiler": "11.2.7",
+    "@angular/core": "11.2.7",
```

Comment:
@jansiegel I have a question to you. After adding these new demos, none of the `package-lock.json` got modified. Is everything alright with `package-lock.json`? For example, in Angular examples, the `basic-example` and `demo` use different versions of Angular. Please advise if something is broken or should be immediately improved now.

Reply:
The `package-lock.json` files inside `examples/[version]/docs/[framework]` directories are generated when running the `examples:install [version]` script from the root of the repo. What it does, is basically:
1. Clears any existing `package-lock` files and `node_modules` directories:
    ```
    Removing:
      11.1.0/**/(js|angular|react|vue)/node_modules
      11.1.0/**/(js|angular|react|vue)/**/node_modules
      11.1.0/**/(js|angular|react|vue)/**/dist
      11.1.0/**/(js|angular|react|vue)/**/.cache
      11.1.0/**/(js|angular|react|vue)/package-lock.json
    ```
    <sup>Excerpt from the log after calling `examples:install [version]`</sup>
2. Runs `npm i --no-audit` in all of the framework-based directories, so currently in:
    - `[version]/docs/angular`
    - `[version]/docs/react`
    - `[version]/docs/vue`
    - `[version]/docs/js`
3. Calls the `./scripts/link-packages.mjs` script to symlink some dependencies, if there's such a need (for examples, for the angular examples)

The `package-lock.json` files inside the example directories are a side-effect of running the examples manually, by calling `npm i` for each of the examples, so they serve another purpose.

I'm not sure if we should be committing any of these, to be honest, maybe it'd be a good idea, to add them to `.npmignore`.

Reply:
OK, I get it. I added the single example's lockfiles to `.gitignore` and explained that in `README.md`. See 523a0ee71c0f0f9c118a0b618c6521a7f5e7298a. WDYT?